Gruntfile.js

```diff
@@ -30,7 +30,7 @@ module.exports = function (grunt) {
                 dest  : 'tests/test262',
                 src   : [
                     'LICENSE',
-                    'test/intl402/*.js',
+                    'test/intl402/**/*.js',
```

Comment:
Added some directories in [test262/test/intl402](https://github.com/tc39/test262/tree/master/test/intl402) so this pattern should be `**/*.js`.